Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/ComponentEditor.hxx

```diff
@@ -101,7 +101,15 @@ namespace AzToolsFramework
 
         void SetComponentOverridden(const bool overridden);
 
-        void EnteredComponentMode(const AZStd::vector<AZ::Uuid>& componentModeTypes);
+        enum class ComponentEditorState
+        {
+            ComponentCardSelected,
+            ComponentCardDisabled
+        };
```

Comment:
Naming suggestion, take it or leave it:
"ComponentEditorState" -> "ComponentModeActiveState". 

(Because this enum is specifically used to say whether or not the component card is active and editable during Component Mode, and doesn't reflect the state of the ComponentEditor at any other time)

Reply:
I took it :)